priv/data/filings_atom_feed.xml

```diff
@@ -7,16 +7,6 @@
 <author><name>Webmaster</name><email>webmaster@sec.gov</email></author>
 <updated>2016-02-28T14:16:53-05:00</updated>
 <entry>
-<title>10-K - Spirit Realty Capital, Inc. (0001308606) (Filer)</title>
-<link rel="alternate" type="text/html" href="http://www.sec.gov/Archives/edgar/data/1308606/000130860616000201/0001308606-16-000201-index.htm"/>
-<summary type="html">
- &lt;b&gt;Filed:&lt;/b&gt; 2016-02-26 &lt;b&gt;AccNo:&lt;/b&gt; 0001308606-16-000201 &lt;b&gt;Size:&lt;/b&gt; 121 MB
-</summary>
-<updated>2016-02-26T18:06:35-05:00</updated>
-<category scheme="http://www.sec.gov/" label="form type" term="10-K"/>
-<id>urn:tag:sec.gov,2008:accession-number=0001308606-16-000201</id>
-</entry>
-<entry>
```

Comment:
Removed this entry because it's huge. 121 megs would take forever to read. Maybe it's worth scanning the rest and removing entries that are too big.